packages/ui/src/views/layouts/PullRequestLayout.tsx

```diff
@@ -21,105 +36,70 @@ enum PullRequestTabsKeys {
   CHANGES = 'changes'
 }
 
-const PullRequestLayout: FC<PullRequestLayoutProps> = ({
+export const PullRequestLayout: FC<PullRequestLayoutProps> = ({
   usePullRequestStore,
   useTranslationStore,
   spaceId,
   repoId,
   updateTitle
 }) => {
-  const { NavLink, Outlet } = useRouterContext()
+  const { Outlet } = useRouterContext()
   const { pullRequest } = usePullRequestStore()
   const { t } = useTranslationStore()
+  const navigate = useNavigate()
+  const location = useLocation()
+
+  const getIsActiveTab = useCallback(
+    (tab: string) => (location.pathname.endsWith(tab) ? 'active' : 'inactive'),
+    [location.pathname]
+  )
+  const makeHandleTabChange = useCallback((tab: string) => () => navigate(tab), [navigate])
```

Comment:
Pretty sure you can just kill this function entirely and just use `navigate` directly

Reply:
Fixed